package.json

```diff
@@ -8,21 +8,23 @@
   "scripts": {
     "build": "yarn run clean && yarn run prep && NODE_ENV=production webpack --progress --colors",
     "clean": "rm -rf build/",
-    "chimp-test": "chimp chimp.config.js",
-    "chimp-test:watch": "yarn run chimp-test --watch",
+    "chimp-test": "concurrently --kill-others --success first \"NODE_ENV=test yarn run dev\" \"chimp chimp.config.js\"",
```

Comment:
Also, want to be explicit about the `--kill others --success first` params passed to `concurrently` as they are not things I had used before.

Basically, by default `concurrently` will continue running as long as any of its child processes have not exited. In our case, `yarn run dev` never exits (unless the bundle fails), so even after our chimp tests complete the `concurrently` process would never exit and our build would not pass. 

So, the `--kill-others` flag partially solves that problem by killing all processes once one finishes (in our case the `chimp` tests). Secondly, the `--success first` tells `concurrently` to exit with a success or failure based on the success/failure of the first process to exist. If we don't use this, `concurrently` will always exit w/ a non-zero code.